newIDE/app/src/Debugger/Inspectors/VariablesContainerInspector.js

```diff
@@ -37,22 +44,33 @@ const transform = variablesContainer => {
   return mapValues(variablesContainer._variables.items, transformVariable);
 };
 
-const handleEdit = (edit, { onCall, onEdit }: Props) => {
+const constructPath = (editPath: string[]) => {
+  const path = [`get(${editPath.shift()})`];
+
+  let skip = false;
+  for (const variableName of editPath) {
+    // Skip every second key as it is the "value"
+    // key which is displayed only for better user
+    // experience but doesn't really exists
+    skip = !skip;
+    if (skip) continue;
+
+    path.push(`getChild(${variableName})`);
```

Comment:
As you get the `variablesContainer` from the props, would it be possible that instead of using `getChild(...)`, you go through the variablesContainer, at the same time as you "walk" down the variable names, so that each time you can check if:
- it's a structure (use `._children`)
- it's an array (use `.childrenList`)
- it's a primitive (something weird is going on, consider aborting).

This would seem to me safer and more aligned with the other way (`transformVariable`) which is also testing the `_type`.

Also it's 100% sure that at some point someone will use a name of a structure to store arbitrary string (for example to do a count of some words or sentences), which will end up having a pair of parenthesis, which will make the debugger crash because of trying to call it as a function.

Let's give no chance to this, and use the `_type` of the variable that can be found by browsing the container, variable by variable :)